l10n_it_location_nuts/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,20 @@
+**Italiano**
+
+Dopo aver installato il modulo, aprire il wizard:
+
+Vendite / Configurazione / Contatti / Localizzazione / Import NUTS 2013
+
+e confermare.
+
+Questo aggiornerà le regioni NUTS con i dati presenti nel `RAMON <http://ec.europa.eu/eurostat/ramon>`__ (Reference And Management Of Nomenclatures)
+e le collegherà alle province già presenti in Odoo.
+
+**English**
+
+After installing the module, open the wizard:
+
+Sales / Configuration / Contacts / Localization / Import NUTS 2013
```

Comment:
Il wizard si trova in Contacts / Configuration / Localization / Import NUTS 2013